rasa/nlu/featurizers/dense_featurizer/convert_featurizer.py

```diff
@@ -32,9 +33,7 @@
 
 # URL to the old remote location of the model which
 # users might use. The model is no longer hosted here.
-ORIGINAL_TF_HUB_MODULE_URL = (
-    "https://github.com/PolyAI-LDN/polyai-models/releases/download/v1.0/model.tar.gz"
-)
+ORIGINAL_TF_HUB_MODULE_URL = "https://github.com/davidalami/ConveRT/releases/download/1.0/nocontext_tf_model.tar.gz"
```

Comment:
this must not be changed, the value is used in the component to inform people to not use this model url anymore